app/src/main/res/values/dimens.xml

```diff
@@ -577,4 +577,9 @@
   <dimen name="profile_reset_pin_activity_profile_reset_input_confirm_pin_margin_end">28dp</dimen>
   <dimen name="profile_reset_pin_activity_profile_reset_save_button_margin_end">28dp</dimen>
 
+  <!-- SectionTitleFragment -->
+  <dimen name="section_title_divider_view_layout_marginTop">32dp</dimen>
```

Comment:
Please rename all these variables correctly to fully camel_case format.
i.e.
`section_title_divider_view_layout_margin_top`

Similar for other files too. Thanks.